bokehjs/src/coffee/mapper/linear_color_mapper.coffee

```diff
@@ -70,12 +70,14 @@ class LinearColorMapper extends HasProperties
 
   _build_palette: (palette) ->
     new_palette = new Uint32Array(palette.length+1)
-    for i in [0...palette.length]
-      if _.isNumber(palette[i])
-        new_palette[i] = palette[i]
+    _convert = (value) ->
+      if _.isNumber(value)
+        return value
       else
-        new_palette[i] = parseInt(palette[i].slice(1), 16)
-    new_palette[new_palette.length-1] = palette[palette.length-1]
+        return parseInt(value.slice(1), 16)
+    for i in [0...palette.length]
+      new_palette[i] = _convert palette[i]
+    new_palette[new_palette.length-1] = _convert palette[palette.length-1]
```

Comment:
@bryevdv I've amended the commit as 3e3c392 (and amended my whining out of the commit message).